src/utils/ddb/ddb_commands.c

```diff
@@ -990,3 +991,82 @@ int ddb_run_dtx_act_abort(struct ddb_ctx *ctx, struct dtx_act_abort_options *opt
 	dtx_modify_fini(&args);
 	return rc;
 }
+
+int
+ddb_run_feature(struct ddb_ctx *ctx, struct feature_options *opt)
+{
+	int      rc;
+	uint64_t new_compat_flags;
+	uint64_t new_incompat_flags;
+	bool     close = false;
+
+	if (ddb_pool_is_open(ctx)) {
+		if (!ctx->dc_write_mode && !opt->show_features)
+			return -DER_INVAL;
+		goto skip;
```

Comment:
What will happen if the pool is opened without `-w` option, but some `{set,clear}_{in}compat_flags` is valid? What is the expected behavior?

Reply:
it will return -DER_INVAL? maybe should return NOPERM.